extensions/interactions/DragAndDropSortInput/directives/oppia-interactive-drag-and-drop-sort-input.component.spec.ts

```diff
@@ -82,12 +90,9 @@ describe('Drag and drop sort input interactive component', () => {
       return {
         previousIndex: previousIndex,
         currentIndex: currentIndex,
-        item: undefined,
-        container: undefined,
-        previousContainer: undefined,
         isPointerOverContainer: true,
         distance: { x: 0, y: 0 }
-      };
+      } as CdkDragDrop<T[], T[]>;
```

Comment:
Can we not use this typecasting here?

Reply:
Again allow us to use typecasting in test files only.